.gitignore

```diff
@@ -2,11 +2,14 @@
 /*.sln.*
 /*.suo
 /*.saved
-/*.cmd
+
+/nuget_modules/
 
 bin/
 obj/
-packages/
+build/
 
 *.user
 *.dotCover
+
+Pandell.PackageRunner.pub
```

Comment:
Please remove. `pub` file should not be generated in the source directory, it should to to `obj` directory that is already ignored.